packages/block-library/src/classic/editor.scss

```diff
@@ -117,6 +118,111 @@
 		border: 2px dashed rgb(186, 186, 186);
 	}
 
+	/* Remove blue highlighting of selected images in WebKit */
+	img[data-mce-selected]::selection {
+		background-color: transparent;
+	}
+
+	/* Image resize handles */
+	.mce-content-body div.mce-resizehandle {
+		border-color: $dark-gray-800;
+		width: 6px;
+		height: 6px;
+	}
+
+	.mce-content-body img[data-mce-selected] {
```

Comment:
> Is the class .mce-content-body needed for...

That class is added to the editor "body" by TinyMCE. It will be there no matter if we use it or not.

Reply:
Let's removed it?

Reply:
These styles didn't work, it should probably be `&.mce-content-body`. I removed them as the default TinyMCE styles seem to be sufficient.